packages/@aws-cdk/aws-ses/lib/configuration-set-event-destination.ts

```diff
@@ -0,0 +1,279 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sns from '@aws-cdk/aws-sns';
+import { IResource, Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { IConfigurationSet } from './configuration-set';
+import { CfnConfigurationSetEventDestination } from './ses.generated';
+
+/**
+ * A configuration set event destination
+ */
+export interface IConfigurationSetEventDestination extends IResource {
+  /**
+   * The ID of the configuration set even destination
+   *
+   * @attribute
+   */
+  readonly configurationSetEventDestinationId: string;
+}
+
+/**
+ * Options for a configuration set event destination
+ */
+export interface ConfigurationSetEventDestinationOptions {
+  /**
+   * A name for the configuration set event destination
+   *
+   * @default - a CloudFormation generated name
+   */
+  readonly configurationSetEventDestinationName?: string;
+
+  /**
+   * Whether Amazon SES publishes events to this destination
+   *
+   * @default true
+   */
+  readonly enabled?: boolean;
```

Comment:
Is there a use case for adding the destination and not enabling it? This seems unnecessary but maybe I'm missing something.

Reply:
Can be used in staging/dev/test environments where it can be "manually" enabled for testing purposes (so you don't pay for 100% of the time enabled in those environments).